cpp/ql/src/semmle/code/cpp/Class.qll

```diff
@@ -912,7 +966,14 @@ abstract class ClassTemplateSpecialization extends Class {
 }
 
 /**
- * A full specialization of a class template.
+ * A full specialization of a class template.  For example:
+ * ```
+ * template<>
+ * class MyTemplateClass<int>
+ * {
+ *   ...
+ * };
+ * ```
```

Comment:
I think this example ought to include the template it's specializing, so that it's self-contained.

Reply:
Done.